### ISSUE TITLE
sysinfo not showing correct mem?

### ISSUE THREAD

**Issue by eyberg** [nanovms/nanos#1141]
```
eyberg@box:~/t$ cat main.c
#include <stdio.h>
#include <sys/sysinfo.h>

int main() {
  struct sysinfo info;

  if (sysinfo(&info) < 0)
    return 0;

  printf("free: %lu\t total: %lu\n", info.freeram, info.totalram);
}
```

**Comment by wjhun** [comment]
Yes, some page.c changes a while back broke physical heap total / allocated reported. It's fixed in now #1140.

**Comment by wjhun** [comment]
Resolved in #1140
